modules/odc-creating-a-project-using-the-developer-perspective-in-the-web-console.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-creating-a-project-using-the-developer-perspective-in-the-web-console_{context}"]
+= Creating a project using the Developer perspective in the web console
+
+You can use the Developer perspective in the {product-title} web console to create a project in your namespace.
+
+.Prerequisites
+
+* Ensure that you have the appropriate roles and permissions to create projects, applications, and other workloads in {product-title}. See link:https://docs.openshift.com/container-platform/3.4/architecture/additional_concepts/authorization.html#roles[here] for more details on roles and permissions.
```

Comment:
We should avoid "here" statements when linking; maybe just have:
xref:../architecture/additional_concepts/authorization.adoc#roles[Learn more about roles and permissions].

Also, note the link format of an xref with a .adoc instead of .html